diff --git a/app/build.gradle b/app/build.gradle
index 3a62b3e..0dde632 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -4,7 +4,7 @@ plugins {

 // 多渠道打包配置已引用的方式传递
 apply from: rootProject.file('flavors.gradle')
-apply from: rootProject.file('signings.gradle')
+// apply from: rootProject.file('signings.gradle')
 apply from: rootProject.file('buildRelease.gradle')

 // By appending the version name, build type, and a timestamp to the APK filename,
@@ -41,22 +41,31 @@ android {
         versionName siyuanVersionName
     }

+    signingConfigs {
+        release {
+            storeFile = file("../keystore/siyuan.jks")
+            storePassword System.getenv("KEYSTORE_PASSWORD")
+            keyAlias "siyuan"
+            keyPassword System.getenv("KEYSTORE_PASSWORD")
+        }
+    }
+
     buildTypes {
         release {
             // 正式版打包时指定签名证书
-            signingConfig signingConfigs.siyuanConfig
+            signingConfig signingConfigs.release

             minifyEnabled true
             shrinkResources true
-            // 此处优先级太高，渠道处配置 resValue 不会生效，故放到了 productFlavors 多渠道打包处处理
-            // resValue "string", "app_name", "SiYuan"
-            resValue "string", "app_package_name", "org.b3log.siyuan"
+            resValue "string", "app_name", "SiYuan"
+            resValue "string", "app_package_name", "org.b3log.siyuan.unlock"
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
         debug {
+            signingConfig signingConfigs.debug
             applicationIdSuffix ".debug"
             minifyEnabled false
-            resValue "string", "app_name", "SiYuan-Debug"
+            resValue "string", "app_name", "SiYuan"
             resValue "string", "app_package_name", "org.b3log.siyuan.debug"
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
