diff --git a/kernel/conf/appearance.go b/kernel/conf/appearance.go
index 6c79d4d..0ff0a64 100644
--- a/kernel/conf/appearance.go
+++ b/kernel/conf/appearance.go
@@ -64,8 +64,8 @@ func NewAppearance() *Appearance {
 		Icon:                "litheness",
 		CodeBlockThemeLight: "github",
 		CodeBlockThemeDark:  "base16/dracula",
-		Lang:                "en",
-		CloseButtonBehavior: 0,
+		Lang:                "zh_CN",
+		CloseButtonBehavior: 1,
 		HideToolbar:         true,
 		HideStatusBar:       false,
 		StatusBar:           util.NewStatusBar(util.IsMobileContainer()),
diff --git a/kernel/conf/sync.go b/kernel/conf/sync.go
index 2fdeb79..3156dad 100644
--- a/kernel/conf/sync.go
+++ b/kernel/conf/sync.go
@@ -39,8 +39,8 @@ func NewSync() *Sync {
 		Enabled:             false,
 		Perception:          false,
 		Mode:                1,
-		GenerateConflictDoc: false,
-		Provider:            ProviderSiYuan,
+		GenerateConflictDoc: true,
+		Provider:            ProviderS3,
 		Interval:            30,
 		LAN:                 &LANSync{MaxConcurrentReqs: 16},
 	}
