diff --git forkSrcPrefix/kernel/api/setting.go forkDstPrefix/kernel/api/setting.go
index 9010350078560bb298de72bc73d54e51db60f21b..dfe9ab286f50431bdb30c633b8e63886b5046419 100644
--- forkSrcPrefix/kernel/api/setting.go
+++ forkDstPrefix/kernel/api/setting.go
@@ -586,7 +586,7 @@ func getPublish(c *gin.Context) {
 	}
 }
 
-func getCloudUser(c *gin.Context) {
+func getCloudUserOrigin(c *gin.Context) {
 	ret := gulu.Ret.NewResult()
 	defer c.JSON(http.StatusOK, ret)
 
@@ -608,6 +608,60 @@ func getCloudUser(c *gin.Context) {
 	ret.Data = model.Conf.GetUser()
 }

+func getCloudUser(c *gin.Context) {
+	ret := gulu.Ret.NewResult()
+	defer c.JSON(http.StatusOK, ret)
+
+	if !model.IsAdminRoleContext(c) {
+		return
+	}
+
+	arg, ok := util.JsonArg(c, ret)
+	if !ok {
+		return
+	}
+
+	t := arg["token"]
+	var token string
+	if nil != t {
+		token = t.(string)
+	}
+	model.RefreshUser(token)
+   originUser := model.Conf.GetUser()
+	user := &conf.User{
+			UserId: originUser.UserId,
+			UserName: originUser.UserName,
+			UserAvatarURL: originUser.UserAvatarURL,
+			UserHomeBImgURL: originUser.UserHomeBImgURL,
+			UserTitles: originUser.UserTitles,
+			UserIntro: originUser.UserIntro,
+			UserNickname: originUser.UserNickname,
+			UserCreateTime: "29991231 00:00:00",
+			UserSiYuanProExpireTime: -1,
+			UserToken: originUser.UserToken,
+			UserTokenExpireTime: "32503593600",
+			UserSiYuanRepoSize: 0,
+			UserSiYuanPointExchangeRepoSize: 0,
+			UserSiYuanAssetSize: 0,
+			UserTrafficUpload: 0,
+			UserTrafficDownload: 0,
+			UserTrafficAPIGet: 0,
+			UserTrafficAPIPut: 0,
+			UserTrafficTime: 0,
+			UserSiYuanSubscriptionPlan: 0,
+			UserSiYuanSubscriptionStatus: 0,
+			UserSiYuanSubscriptionType: 1,
+			UserSiYuanOneTimePayStatus: 1,
+	}
+
+	model.Conf.User = user
+	data, _ := gulu.JSON.MarshalJSON(user)
+	model.Conf.UserData = util.AESEncrypt(string(data))
+	model.Conf.Save()
+
+	ret.Data = user
+}
+
 func logoutCloudUser(c *gin.Context) {
 	ret := gulu.Ret.NewResult()
 	defer c.JSON(http.StatusOK, ret)
